pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-slack/default.nix

```diff
@@ -1,14 +1,13 @@
 { stdenv, fetchFromGitHub, pidgin, pkgconfig }:
 
 stdenv.mkDerivation {
-  pname = "purple-slack-unstable";
```

Comment:
`pname` shouldn't be removed.

Reply:
Didn't notice that at all. Fixed in latest commit